core/src/main/res/values/strings.xml

```diff
@@ -621,119 +621,119 @@
-    <string name="gpodnetauth_finish_butsyncnow">Start sync now</string>
+    <string name="gpodnetauth_finish_descr">Your gpodder.net account is now linked with your device. Your subscriptions from gpodder.net will auto-sync to your device.</string>
+    <string name="gpodnetauth_finish_butsyncnow">Sync…</string>
 
-    <string name="gpodnetsync_error_title">gpodder.net sync error</string>
-    <string name="gpodnetsync_error_descr">An error occurred during syncing:\u0020</string>
-    <string name="gpodnetsync_pref_report_successful">Successful</string>
+    <string name="gpodnetsync_error_title">Could not sync subscriptions from gpodder.net</string>
+    <string name="gpodnetsync_error_descr">Could not perform sync:\u0020</string>
+    <string name="gpodnetsync_pref_report_successful">Done</string>
     <string name="gpodnetsync_pref_report_failed">Failed</string>
-    <string name="gpodnetsync_username_characters_error">Usernames may only contain letters, digits, hyphens and underscores.</string>
+    <string name="gpodnetsync_username_characters_error">Only use letters, digits, hyphens and underscores.</string>
 
     <!-- Directory chooser -->
     <string name="choose_data_directory">Choose Data Folder</string>
     <string name="choose_data_directory_message">Please choose the base of your data folder. AntennaPod will create the appropriate sub-directories.</string>
-    <string name="choose_data_directory_available_space">%1$s of %2$s free</string>
-    <string name="pref_pausePlaybackForFocusLoss_sum">Pause playback instead of lowering volume when another app wants to play sounds</string>
+    <string name="choose_data_directory_available_space">%1$s of %2$s available</string>
+    <string name="pref_pausePlaybackForFocusLoss_sum">Pause playback instead of lowering volume for other playback</string>
     <string name="pref_pausePlaybackForFocusLoss_title">Pause for Interruptions</string>
-    <string name="pref_resumeAfterCall_sum">Resume playback after a phone call completes</string>
+    <string name="pref_resumeAfterCall_sum">Resume playback after a phone calls</string>
     <string name="pref_resumeAfterCall_title">Resume after Call</string>
-    <string name="pref_restart_required">AntennaPod has to be restarted for this change to take effect.</string>
+    <string name="pref_restart_required">Restart the app to use the new settings.</string>
 
     <!-- Online feed view -->
     <string name="subscribe_label">Subscribe</string>
-    <string name="subscribing_label">Subscribing&#8230;</string>
+    <string name="subscribing_label">Subscribing…</string>
     <string name="preview_episode">Preview</string>
     <string name="stop_preview">Stop preview</string>
 
     <!-- Content descriptions for image buttons -->
     <string name="rewind_label">Rewind</string>
```

Comment:
Icons/pictograms could be used here. Or added like with the "Plus (+) button" scheme.

Reply:
Which string(s) does your comment refer to? Note that the strings following line 668 are actually just "Content descriptions for image buttons", for those using screen readers.

Reply:
Rewind, fast forward.